### PR TITLE
Home

### PR DESCRIPTION
**Pull request by sara-34**
Uploading page with changes in the home

### CODE REVIEW

File: pagina-practica/practica.html [hunk 1]
```diff
@@ -21,7 +21,55 @@
...
+            <p id="text">Computer Science is the branch<br>
+             of Engineering that studies<br>
+             the hardware, data networks and<br>
+             software necessary to process information<br>
+             automatically.</p><br>
+        </div>
+
+    </article>
+
+    <footer class="contacto" id="contactos">
```
Comment (carlos4rias) [inline pagina-practica/practica.html]: Why you use a footer here, it is really a footer?, also there is not consistency at the lines 29 and 25 why use article and then section respectively

File: pagina-practica/practica.html [hunk 2]
```diff
@@ -21,7 +21,55 @@
...
+        <div id="contacto">
+            <h3>Contacto</h3>
+            <img id="imagen-about"src="../pagina-practica/imagenes/undraw_mobile_analytics_72sr.png" alt="">
+        </div>
+
+        <div class="formulario">
+            
+                <div class="informacion">
```
Comment (carlos4rias) [inline pagina-practica/practica.html:58]: Why did not use the form element?, always is recommended to use the form element for good practices.

File: pagina-practica/practica.html [hunk 3]
```diff
@@ -21,7 +21,55 @@
...
+                    <label for="last name">Last-name:</label>
+                    <input type="Last name" id="last name">
+                
+                    <label for="menssage">Comment:</label>
+                    <textarea id="menssage" name="user_message"></textarea>
+               
+                <div class="button">
+                    <button type="siguiente">Start</button>
+
+                </div>
```
Comment (carlos4rias) [inline pagina-practica/practica.html]: Always ident the code, there is not very clear which div element are you closing also it is a good practice.

File: pagina-practica/style.css [hunk 4]
```diff
@@ -29,3 +29,81 @@
     padding-top: 4px;
 }
 
+#img{
+    display: flex;
+    width: 600px;
+    height: 350px;
+    border: solid;
+    margin-top: 10px;
```
Comment (carlos4rias) [inline pagina-practica/style.css]: use a single `margin` property, for example
```css
margin: 10px 0 10px 0
```

File: pagina-practica/style.css [hunk 5]
```diff
@@ -29,3 +29,81 @@
...
+    font-size: 25px;
+}
+
+.formulario{
+    width: 200px;
+    height: 200px;
+    background: lavender;
+    border: solid;
+    padding: 15px;
+    padding-top: 20px;
```
Comment (carlos4rias) [inline pagina-practica/style.css]: why using `padding` and `padding-top` properties, mix up in a single `padding` property

**Review by carlos4rias — CHANGES_REQUESTED** [review]

File: pagina-practica/practica.html (hunk 1 above)
Comment (sara-34) [reply pagina-practica/practica.html]: making suggestion

File: pagina-practica/style.css [hunk 6]
```diff
@@ -30,12 +30,164 @@
...
+    gap: 100px;
+
+}
+
+.parrafo{
+    display: block;
+    align-items: column;
+    margin: 12px 300px 0 0;
+}
+#imagen-about{
```
Comment (carlos4rias) [inline pagina-practica/style.css]: Why still using ID's as selectors?
https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4 
https://hackernoon.com/why-ids-can-be-problematic-in-css-gt103uh6
https://www.quora.com/Why-is-it-considered-bad-form-to-use-an-ID-selector-in-a-CSS-stylesheet
 read this please.

File: pagina-practica/practica.html [hunk 7]
```diff
@@ -26,5 +26,59 @@
...
+            <p id="text">Computer Science is the branch<br>
+             of Engineering that studies<br>
+             the hardware, data networks and<br>
+             software necessary to process information<br>
+             automatically.</p><br>
+        </div>
+
+    </main>
+
+<div>
```
Comment (carlos4rias) [inline pagina-practica/practica.html]: Why using a div, you are confused with HTML semantic

File: pagina-practica/practica.html [hunk 8]
```diff
@@ -26,5 +26,59 @@
...
+
+
+    <section class="contacto" id="contactos">
```
Comment (carlos4rias) [inline pagina-practica/practica.html:48]: Why a section here?

**Review by carlos4rias — CHANGES_REQUESTED** [review]

**Review by carlos4rias — APPROVED** [review]

**Pull request merged**